app/models/group.rb

```diff
@@ -414,6 +414,16 @@ def bulk_add(user_ids)
       if self.title.present?
         User.where(id: user_ids).update_all(title: self.title)
       end
+
+      if self.grant_trust_level.present?
+        User.where(id: user_ids).find_each do |user|
```

Comment:
That is a valid concern. What should be the correct approach here? A new background job for changing trust level in bulk?